tensorflow/contrib/tensorrt/convert/convert_nodes.cc

```diff
@@ -2633,6 +2856,54 @@ tensorflow::Status ConvertTopK(Converter& ctx,
 }
 #endif
 
+void ApplyQuantizationRanges(Converter& converter, bool use_calibration) {
+  std::vector<nvinfer1::ITensor*> tensors_missing_ranges;
+  // Infer quantization ranges backward through reshapes, maxpool, etc
+  for (auto it : converter.quantization_infer_backwards_) {
+    if (converter.quantization_scales_.find(it.first) != converter.quantization_scales_.end()) {
+      // LOG(INFO) << "infer backwards" << it.second->getName() << " <- " << it.first->getName() << "(" <<  converter.quantization_scales_[it.first] << ")";
+      // LOG(INFO) << "                " << static_cast<void*>(it.second) << " <- " << static_cast<void*>(it.first);
+      converter.quantization_scales_[it.second] = converter.quantization_scales_[it.first];
+    }
+    else {
+      tensors_missing_ranges.push_back(it.first);
+    }
+  }
+  // Get all tensors from network
+  std::vector<nvinfer1::ITensor*> all_tensors;
```

Comment:
I believe there will be ~50% duplicates in the vector, so maybe use a std::set instead?